custom_components/plugwise/coordinator.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, hass: HomeAssistant, api: Smile, interval: timedelta) -> None
             request_refresh_debouncer=Debouncer(
                 hass,
                 LOGGER,
-                cooldown=1.5,
+                cooldown=5.0,
```

Comment:
Given that this might upset the frontend on lower class devices if we take this route, should we make it configurable?

Reply:
Implemented as suggested.